### ISSUE TITLE
Make governance committee membership replaceable

### ISSUE THREAD

**Issue by Chris-Hibbert** [Agoric/Agoric#5539]
## What is the Problem Being Solved?

We want the ability to update the membership of governance committees.

## Description of the Design

Add a method to the committee's creator facet allowing the holder to replace the committee's members. This could be called by something in the bootstrap context after a cosmos-level vote to elect a new slate of addresses.

## Security Considerations

governance, legibility. 

## Test Plan

check wrong number of nominations. Verify legibility.

**Comment by dckc** [comment]
@Chris-Hibbert this part of the design isn't quite clear to me:

> Add a method to the committee's creator facet allowing the holder to replace the committee's members.

What does it mean to replace the committee's members? Is it a matter of regenerating the voter invitations?

(If it's clear to you, I can stand by until you find time to code it up for an explanation.)

**Comment by Chris-Hibbert** [comment]
The representation inside `committee.js` would have to change. It doesn't currently track the outstanding `voterHandle`s or have a way to invalidate theme.  It would have to remember the voterHandles, check them when voting, and replace them with a new set on changes. (A more OCap-ish approach might be to have an intermediate object that could cast the votes, and be invalidated when replaced.)

The method should probably allow the caller to specify a number of invitations, and return that number of new invitations.

**Comment by dckc** [comment]
> A more OCap-ish approach might be to have an intermediate object that could cast the votes, and be invalidated when replaced.

Yeah... we seem to moving from using a replaceable electorate to using a long-standing electorate with revocable membership.

This design makes sense.

**Comment by dckc** [comment]
I just ran across some related work:
> In accordance with [Prop 25](https://www.mintscan.io/juno/proposals/25) the SubDAO Multisig contract will be instantiated by this governance vote with the Juno community as admin. This means the community can execute messages on behalf of the DAO via a governance proposal, recall funds, upgrade the contract, or add / remove members. -- https://commonwealth.im/juno/discussion/5264-launch-hack-juno-subdao

**Comment by dckc** [comment]
having the committee be able to update its own membership is important.
